quicklisp/config.lisp

```diff
@@ -19,6 +19,22 @@
                      :directory (list* :relative directory-parts))
       base)))
 
+(defun config-paths (&optional (path-string "**/*"))
```

Comment:
I don't want the fact that config values are backed by the filesystem to be part of the API. Wildcards are ok, but tying them to the filesystem syntax and semantics are not.

I'd also be interested in a `config-values` function that is something like `(pairlis config-paths (mapcar 'config-value config-paths))`.